packaging/windows/BUILD.txt

```diff
@@ -6,44 +6,61 @@ A) Native compile using MSYS2:  How to make a darktable windows installer (64 bi
 
     Install MSYS2 (instructions and prerequisites can be found on official website: https://msys2.github.io/).
     Update the base MSYS2 system until no further updates are available using:
-    $ pacman -Syu
+		$ pacman -Syu
 
-    Install x64 developer tools and x86_64 toolchain and git from the MSYS2 prompt:
-    $ pacman -S base-devel
-    $ pacman -S mingw-w64-x86_64-{toolchain,cmake,nsis}
-    $ pacman -S git
+    From MSYS2 terminal, install x64 developer tools and x86_64 toolchain and git from the MSYS2 prompt:
+		$ pacman -S base-devel
+		$ pacman -S mingw-w64-x86_64-{toolchain,cmake,nsis}
+		$ pacman -S git
 
     Install required libraries and dependencies:
-    $ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,dbus-glib,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk3,pugixml,libexif,osm-gps-map,libgphoto2,flickcurl,drmingw,gettext,python3,iso-codes}
+		$ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,dbus-glib,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk3,pugixml,libexif,osm-gps-map,libgphoto2,flickcurl,drmingw,gettext,python3,iso-codes,python3-jsonschema,python2-setuptools,python3-setuptools}
```

Comment:
No need for python2-setuptools: we don't install python 2
IMHO no need for python3-setuptools - can you elaborate why it is needed?

Reply:
Without python3-jsonschema there is a warning in the configuration
`Missing jsonschema, problems in noiseprofiles.json might go unnoticed`

With python3-jsonschema only there is an error in the build

```
[ 92%] Checking validity of noiseprofiles.json
Traceback (most recent call last):
  File "C:\msys64\mingw64\bin\jsonschema-script.py", line 6, in <module>
    from pkg_resources import load_entry_point
ModuleNotFoundError: No module named 'pkg_resources'
make[2]: *** [data/CMakeFiles/validate_noiseprofiles_json.dir/build.make:59: data/CMakeFiles/validate_noiseprofiles_json] Error 1
make[1]: *** [CMakeFiles/Makefile2:8021: data/CMakeFiles/validate_noiseprofiles_json.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
```

The missing module pkg_resources is provided by the package python3-setuptools.
The package python2-setuptools is not needed you are right.

Reply:
I see. Actually seems to be an issue with the packaging of python3-jsonschema, and needs to be fixed there. Anyhow leave then the python3-setuptools here for the time being.